provisioning/miniprov/hare_mp/main.py

```diff
@@ -1009,6 +1031,57 @@ def main():
         help='Target directory; defaults to /tmp/hare.',
         action='store')
 
+    sb_sub_parser.add_argument(
+        '--duration',
+        type=str,
+        nargs=1,
+        help='Time bound log collection, like logs for last n days /'
```

Comment:
Can we specify the syntax of the duration argument?

Reply:
Yes Sure. Added syntax of this.